### PR TITLE
chore: migrate emotes to new function

### PR DESCRIPTION
**Pull request by kevmo314**

### CODE REVIEW

File: test/models/messages/twitch/message_test.dart [hunk 1]
```diff
@@ -114,21 +114,28 @@ void main() {
     });
 
     test('third party emotes should tokenize', () {
-      final source = Uri.parse("https://3pemote");
+      final source = "https://3pemote";
```
Comment (juaoose) [inline test/models/messages/twitch/message_test.dart:117]:
```suggestion
      const source = "https://3pemote";
```

File: test/models/messages/twitch/message_test.dart [hunk 2]
```diff
@@ -199,18 +206,25 @@ void main() {
     });
 
     test('third party emotes and emojis tokenize', () {
-      final source = Uri.parse("https://3pemote");
+      final source = "https://3pemote";
```
Comment (juaoose) [inline test/models/messages/twitch/message_test.dart:209]:
```suggestion
      const source = "https://3pemote";
```

**Review by juaoose — APPROVED** [review]

**Comment by juaoose** [comment]
Looks like we either have to grab the small emote for some of the 3rd party ones or limit the size of the emotes when displayed.
Screenshot_20220906-133502

**Comment by kevmo314** [comment]
> Looks like we either have to grab the small emote for some of the 3rd party ones or limit the size of the emotes when displayed. Screenshot_20220906-133502

Fixed in https://github.com/muxable/rtchat/pull/679

That looks hilarious though maybe we should leave it as an easter egg

**Pull request merged**